### PR TITLE
[REM] stock_picking_batch: remove choose line widget

### PR DESCRIPTION
**Pull request by mama-odoo**
Description of the issue/feature this PR addresses:
Introduced in https://github.com/odoo/odoo/commit/dd53f5323f5f582152a858d61990c99677ed1351#diff-3d5851f58ca78be95289219721c025e0ab416d525f2d4f95a13bc2740f7b226f, the choose line widget is never used and can be safely removed.

Current behavior before PR:
The file choose_lines_widget.js is never called/used

Desired behavior after PR is merged:
Cleaning

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock_picking_batch/__manifest__.py [hunk 1]
```diff
@@ -27,11 +27,6 @@
     'demo': [
         'data/stock_picking_batch_demo.xml',
     ],
-    'assets': {
```
Comment (amoyaux) [inline addons/stock_picking_batch/__manifest__.py:30]: Use a proper commit message.

Guidelines are always
[TAG] module_name: small description
[DEL] stock_picking_batch: remove choose line widget

Then complete the description.
Link to the commit that introduce the widget and explain why you can remove it. (In this case it's easy, just say that he was not use)

**Comment by amoyaux** [comment]
robodoo r+

**Comment by amoyaux** [comment]
robodoo r-

**Comment by amoyaux** [comment]
robodoo r+

**Comment by amoyaux** [comment]
robodoo r+

**Comment by Whenrow** [comment]
robodoo r+

**Comment by amoyaux** [comment]
robodoo r+

**Pull request closed without merging**